src/status_im/navigation/options.cljs

```diff
@@ -81,6 +81,7 @@
 
 (def transparent-screen-options
   (merge
+   (statusbar-and-navbar-root)
```

Comment:
We are overriding `statusBar` key in the end. So I am little confused, how this is fixing the issue.

Reply:
For these screens, it doesn't override the option.

Reply:
I meant, we just have here
```
(merge
{:statusBar ...}
{:statusBar ...})
```

Reply:
actually not status bar, it's for navigation bar.

Reply:
Thank you for clearing. As https://github.com/status-im/status-mobile/issues/17636 was highlighting issues related to status-bar I confused that.

Should we just remove `statusBar` key then? As we are already merging with above function

Reply:
Yeah, I don't think we need `statusBar` key.

Reply:
Cool, please remove additional `statusBar` key then.